examples/simulation/Tutorial1/src/FairTutorialDet1.h

```diff
@@ -83,6 +83,8 @@ class FairTutorialDet1: public FairDetector
 
     virtual FairModule* CloneModule() const;
 
+    virtual Bool_t      CheckIfSensitive(std::string name);
```

Comment:
perhaps simply `IsSensitive` is a more concise name?
E.g.: `if (pixelDetector.IsSensitive()) { ... }`. Ignore this if the name has to be like you have it...